src/Middleware/Payload.php

```diff
@@ -28,6 +33,15 @@ public function __construct(array $options = [])
     }
 
     /**
+     * If the Request object already has a parsedBody, normally Payload will skip parsing. This is not always
+     * desirable behavior. Calling this setter allows you to override this behavior.
+     */
+    public function overrideExistingParsedBody()
```

Comment:
All methods for boolean options has the same signature `option($option = true)` because it's more readable (IMHO), keeps an API consistency across all middleware components and not always the default value is `false`. For example, if in the future we decide to make `override` true by default, the user can disable it easily. Other advantage is this allows to set the configuration options in other place. For example, instead this:

```php
$middleware = Middleware::Payload();

if ($config['payload:override']) {
    $middleware->override();
}
```

You can do simply this:

```php
$middleware = Middleware::Payload()->override($config['payload:override']);
```